Improve authorization doc structure (#1024)
SAP/luigi: docs/authorization-configuration.md

# Authorization configuration

To configure authorization in Luigi:

1. Configure the [general authorization options](#general-authorization-options).

2. Choose an authorization provider:
    * [OpenID Connect](#openid-connect-configuration)
      * [Third-party cookies and silent token refresh](#third-party-cookies-and-silent-token-refresh)
    * [OAuth2 Implicit Grant](#oauth2-implicit-grant-configuration)
    * [Custom authorization provider](#custom-authorization-provider)
    * [Create your own authorization provider](#implement-a-custom-authorization-provider)

## General authorization options

<!-- accordion:start -->
### How do I configure authorization in Luigi?

> Luigi authorization can be configured the `auth:` section of your Luigi configuration file. To see how authorization works, you can also go to the [Luigi Fiddle](https://fiddle.luigi-project.io) site and configure a sample application.

Reviewer: ```suggestion
You can configure Luigi authorization using the `auth:` section of your Luigi configuration file. To see how authorization works, go to the [Luigi Fiddle](https://fiddle.luigi-project.io) site and configure a sample application.
```
I would use active voice in both cases.

}
```

The **use** key defines the active authorization provider and the **disableAutoLogin** key allows you to disable the automatic login flow that is provided by default.

### How do I show some navigation nodes only to non-authenticated users?

> To show certain nodes only to non-authenticated users, utilize the [anonymousAccess](navigation-parameters-reference.md#anonymousaccess) parameter.

Reviewer: ```suggestion
To show certain nodes only to non-authenticated users, use the [anonymousAccess](navigation-parameters-reference.md#anonymousaccess) parameter.
```
I would go with `use` :)
